src/frontends/sql/BUILD

```diff
@@ -37,7 +37,9 @@ cc_proto_library(
 
 cc_library(
     name = "decoder_context",
+    srcs = ["decoder_context.cc"],
     hdrs = ["decoder_context.h"],
+    visibility = [":__subpackages__"],
```

Comment:
I need it so that the `MergeOperationView` in the `testing` subpackage can have access to the string constants on `DecoderContext`. `MergeOperationView` doesn't need to see anything else from the `//src/frontends/sql` package though.